docs/python/debugging.md

```diff
@@ -125,25 +328,13 @@ You can also rely on a relative path from the workspace root. For example, if th
 "program": "${workspaceFolder}/pokemongo_bot/event_handlers/__init__.py",
 ```
 
-### `pythonPath`
+### `python`
```

Comment:
This needs to mention that the property value can be an array instead of a simple string, which allows passing arguments to the Python interpreter, e.g.:
```json5
"python": ["/usr/bin/python3", "-O"]
```

Reply:
Thanks, added the following to that section. Look ok?

If you need to pass arguments to the Python interpreter, you can use the syntax `"python": ["<path>", "<arg>",...]`.